libs/application/template-api-modules/src/lib/modules/templates/accident-notification/accident-notification.service.spec.ts

```
@@ -17,15 +23,11 @@ import { ApplicationService } from '@island.is/application/api/core'
 import { AccidentreportsApi } from '@island.is/clients/icelandic-health-insurance/rights-portal'
 import { createApplication } from '@island.is/application/testing'
-import { createCurrentUser } from '@island.is/testing/fixtures'
 import get from 'lodash/get'
 import set from 'lodash/set'
-import type { Locale } from '@island.is/shared/types'
-import {
-  ApplicationStatus,
-  ApplicationTypes,
-} from '@island.is/application/types'
-
+import { S3Service } from './attachments/s3.service'
```

Biome

[error] Shouldn't redeclare 'S3Service'. Consider to delete it or rename it.

'S3Service' is defined here:


(lint/suspicious/noRedeclare)